puppet/modules/quickstack/manifests/cinder_volume.pp

```diff
@@ -223,37 +248,59 @@
       }
     }
 
-    if str2bool_i("$backend_eqlx") {
+  if str2bool_i($backend_dell_sc) {
+
+      $count = size($backend_dell_sc_name)
+      $last = $count -1
+      $dell_sc_backends = produce_array_with_prefix('dell_sc',1,$count)  #Initialize with section headers
+      
+      quickstack::dellsc::volume { $last:
+        index                          => $last,
+        backend_section_name_array     => $dell_sc_backends,
+        backend_dell_sc_name_array     => $backend_dell_sc_name,
```

Comment:
Do not recall seeing backend_xxx parameters defined in previous files like seed.rb and xxx.yml.